backend/src/app.js

```diff
@@ -14,37 +14,43 @@ import MessageTriage from './lib/message-triage'
 const app = express()
 app.disable('x-powered-by')
 app.use(function (_req, res) { res.send({ msg: 'WebSocket Only!' }) })
-app.use(morgan('combined', { stream: winston.stream }))
+app.use(morgan('combined', { stream: logger.stream }))
 
 const server = http.createServer(app)
 const socketio = io(server, { pingTimeout: 30000 })
+const genricMessage = 'message'
+const mopidyMessage = 'mopidy'
 
 MongodbService()
-
-MopidyService(socketio, mopidy => {
-  if (mopidy.playback) {
+MopidyService(
+  (data) => socketio.emit(genricMessage, data),
+  (online) => socketio.emit(mopidyMessage, Payload.toJsonString({ online })),
+  (mopidy) => {
```

Comment:
Since these three functions (being passed to `MopidyService` as arguments) are anonymous, it's not obvious what they are.

I think it'd be clearer if they were declared separately and named, and then passed to `MopidyService()` afterwards:

```js
const broadcastToAll = (data) => socketio.emit(genricMessage, data)

const mopidyState = socketio.emit(mopidyMessage, Payload.toJsonString({ online }))

const cbAllowConnections = (mopidy) => { ... }

MopidyService(broadcastToAll, mopidyState, cbAllowConnections)
```